readme.txt

```diff
@@ -1,448 +1,475 @@
-* Bug fix: Editorial comments should no longer show up in the stock Recent Comments widget or in the comments view in the WordPress Admin. The comment count number should also be correct.
-* Bug fix: Duplicate custom post statuses and usergroups are handled in more sane ways (aka creating, editing, and deleting should work as expected)
-
-= 0.5.3 (Oct. 6, 2010) =
-* Fixes issue where default Custom Statuses and User Groups were returning even after being deleted
-
-= 0.5.1 (Jul. 29, 2010) =
-* Editorial calendar improvements: filter by category or author
-* QuickPitch stories get default status instead of pitch status
-* No email notifications for “Auto Draft” post status
-* Backwards compatibility with WordPress 2.9.x
-
-= 0.5 (Jul. 3, 2010) =
-* Calendar view for visualizing and spec assignments at a glance
-* Improvements for WordPress 3.0 compatibility
-
-= 0.4 =
-* Users that edit a post automatically get subscribed to that post (only if they have the manage subscriptions capability)
-* Edit Flow automatically hides editorial comments if the plugin is disabled
-* Moved default custom status additions to upgrade function so they don't get added every time you activate
-* Bug fix: remove editorial comments from comments feed
-
-= 0.3.3 (Feb. 4, 2010) =
-* Added tooltips with descriptions to the Status dropdown and Status Filter links. Thanks to [Gil Namur](http://lifeasahuman.com) for the great idea!
-* Fixed the issue where subscribed users/usergroups were not receiving notifications
-
-= 0.3.2 (Jan. 28, 2010) =
-* Fixed fatal error if notifications were disabled
-
-= 0.3.1 =
-* Small bug fixes
-
-= 0.3 =
-* *Note:* Edit Flow now requires 2.9+
-* Notification emails on status change now have specific subject lines messages based on action taken
-* Action links in comment notifications now take the user to the comment form; i.e. clicking reply link in the email will focus on the comment text box and reply to the message
-* Usergroups!
-* Assign users and usergroups that should be notified of post updates
-* Removed notify by role option since it's redundant because of usergroups
-* Added "Always notify admin option"
-* Added option to hide the status dropdown on Post and Page edit pages (default set to show)
-* Added option to globally disable QuickPitch widget
-* Bug fix: Custom Status names cannot be longer than 20 chars
-* Bug fix: Deleted users are removed as subscribers from posts
-* Bug fix: Blank menu items should now be sorta hidden
-
-= 0.2 =
-* Custom Statuses are now supported for pages
-* Editorial Comments (with threading)
-* Email Notifications (on post status change and editorial comment)
-* Additional Post metadata
-* Quick Pitch Dashboard widget
-* Bug fix: sorting issue on Manage Posts page (Mad props to David Smith from Columbia U.)
-* Other bug fixes
-* Better localization support
-
-= 0.1.5 =
-* Ability to assign custom statuses to posts
+=== Edit Flow ===

```

Comment:
I had this happen as well, and the way I avoided the entire file changing was by doing cmd+k followed by s in vscode. That way it didn't format this file. Prob worth adding it to the prettierignore maybe

Reply:
On second thoughts, lets leave it so we don't need to keep doing this revert. Better to get it done in one go